scripts/export-meshseg-onnx.py

```
@@ -451,6 +451,15 @@ def make_tree(rng):
     top = np.array([lean[0] * trunkH, trunkH, lean[1] * trunkH])
     parts.append((lambda n, b=top, r=trunkR: capsule_surf([0, 0, 0], b, r, n, rng),
                   TRUNK, trunkH * trunkR * 3))
+    # Trunk base FLARE / buttress — most real trunks widen at the bottom. This is
+    # TRUNK, not root: without it the model learned "wide low mass = root" and
+    # grabbed the trunk flare (user report). A short, wide, low cone at the base.
+    if rng.random() < 0.7:
+        flareR = trunkR * rng.uniform(1.6, 3.5)
+        flareH = trunkH * rng.uniform(0.05, 0.18)
+        parts.append((lambda n, top=np.array([0.0, flareH, 0.0]), r=flareR:
```

Ruff (0.15.21)

[warning] Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)